templates/zerver/help/quote-and-reply.md

```diff
@@ -16,16 +26,15 @@ start a new topic.
 
 {end_tabs}
 
-You can also use the keyboard shortcut `>` to initiate quote-and-reply.
+!!! keyboard_tip ""
```

Comment:
Let's change:

> (optional) Delete paragraphs not relevant to your reply, just like
   when replying to a multi-paragraph email.

To something like: 

> (optional) Delete any parts of the quoted message that are not relevant to your reply.

I don't think we need to mention email at this point, and you don't have to delete whole paragraphs.

Reply:
Yes, makes sense. Done in update.